docs/guides/code_examples/running_in_web_server/crawler.py

```diff
@@ -0,0 +1,54 @@
+import asyncio
+from collections.abc import AsyncIterator
+from contextlib import asynccontextmanager
+from typing import TypedDict
+
+from fastapi import FastAPI
+
+from crawlee.crawlers import ParselCrawler, ParselCrawlingContext
+
+
+class State(TypedDict):
```

Comment:
Not really important, but in FastAPI, you usually use [dependencies](https://fastapi.tiangolo.com/tutorial/dependencies/) for this kind of business.